app/Http/Controllers/ImportsResultsController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Services\ImportResults;
+
+class ImportsResultsController extends Controller
+{
+    public function showResultsCsv(int $import_id)
+    {
+        $filename = strtr(config('imports.results.filename_template'), [
+            ':datetime' => now()->format('Y-m-d_H-i-s')
```

Comment:
I’d use colons instead of dashes in the “time” part, and preferably also a `T` instead of `_` to separate the “date” and “time” parts, to match ISO 8601.